model/clusters_mgmt/v1/cluster_metrics_type.model

```diff
@@ -35,4 +38,7 @@ struct ClusterMetrics {
 
 	// The number of nodes provisioned for the cluster.
 	Nodes ClusterNodes
+
+    // The operation system provisioned for the cluster.
+	OperatingSystem String
```

Comment:
Do we want to make this a separate enum type? We only support `Red Hat Enterprise Linux` and `Red Hat Enterprise Linux CoreOS`, whereas a string would allow arbitrary values. Not a blocker.

Reply:
We already have a cluster operating system type, is it something you can use here?

Reply:
Yep, good point. I can update it. just to confirm, I happened to review this from @gshilin-sdb ,
https://github.com/openshift-online/ocm-api-model/pull/139/files#diff-bcc75eb265fd6b0141bc43bafaeaedf6R17

dose the enum break right now?

Reply:
I remove `OperatingSystem`. I am sure I m not using it now.